mypy/main.py

```diff
@@ -552,7 +552,7 @@ def add_invertible_flag(flag: str,
         dest='shadow_file', action='append',
         help="When encountering SOURCE_FILE, read and type check "
              "the contents of SHADOW_FILE instead.")
-    add_invertible_flag('--fast-exit', default=False, help=argparse.SUPPRESS,
+    add_invertible_flag('--fast-exit', default=True, help=argparse.SUPPRESS,
```

Comment:
I would say if we enable this by default we should at least make this publicly visible. Also Ideally this not affect runs made through `api.run()` calls.

cc @gvanrossum

Reply:
Forgive me if you find my question stupid but I think I need a bit of clarification here. What exactly do you mean by "make this publicly visible"? As in, should the user be warned that "--fast-exit" is now enabled by default? or is there something I'm unable to understand?
As I've only started contributing recently, any guidance will be much appreciated.

Reply:
> What exactly do you mean by "make this publicly visible"?

Two things:
* Remove `help=argparse.SUPPRESS`
* Describe it in the docs.

Reply:
--fast-exit was never described in the documentation as an experimental feature or otherwise. I've added it :smile: